core/localization/src/main/res/values-zh/strings.xml

```diff
@@ -396,23 +399,23 @@
     <string name="in_paint_tab_2">调整</string>
 
     <!-- 修复模式选项 -->
-    <string name="in_paint_mode_masked">修复遮罩</string>
-    <string name="in_paint_mode_not_masked">未遮罩修复</string>
+    <string name="in_paint_mode_masked">修复掩码</string>
+    <string name="in_paint_mode_not_masked">未掩码修复</string>
 
     <string name="in_paint_mask_content_fill">填充</string>
     <string name="in_paint_mask_content_original">原始</string>
-    <string name="in_paint_mask_content_latent_noise">潜在噪声</string>
-    <string name="in_paint_mask_content_latent_nothing">潜在无</string>
+    <string name="in_paint_mask_content_latent_noise">潜空间噪声</string>
+    <string name="in_paint_mask_content_latent_nothing">潜空间零向量</string>
 
     <string name="in_paint_area_whole">整张图片</string>
-    <string name="in_paint_area_only_masked">仅遮罩</string>
+    <string name="in_paint_area_only_masked">仅掩码</string>
 
-    <string name="donate_title_thanks">感谢这些了不起的人的支持❤</string>
+    <string name="donate_title_thanks">感谢这些了不起的人们提供的支持 ❤</string>
     <string name="donate_bs_title">SDAI 是一款 Android 应用程序，它：</string>
-    <string name="donate_bs_sub_title">• 通过稳定的扩散 AI 为您提供创建图像的能力；\n• 让您自由选择生成提供商；\n• 没有广告、遥测并且不会监视您；\n• 是一个开源软件；\n• 欢迎您免费使用。</string>
-    <string name="donate_bs_ending">如果您发现该软件有价值，并且想要表示感谢并表示一点支持，请点击此处的按钮。</string>
+    <string name="donate_bs_sub_title">• 赋予您使用 Stable Diffusion AI 创建图像的能力； • 让您自由选择生成供应商； • 无广告、无遥测，无监控； • 是一款开源软件； • 欢迎您免费使用。</string>
```

Comment:
This string should contain new line characters `\n` after each phrase before every next `•` symbol.

```suggestion
    <string name="donate_bs_sub_title">• 赋予您使用 Stable Diffusion AI 创建图像的能力；\n• 让您自由选择生成供应商；\n• 无广告、无遥测，无监控；\n• 是一款开源软件；\n• 欢迎您免费使用。</string>
```

Reply:
Right, missed this one.